config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -1555,51 +1555,6 @@ presubmits:
     name: pull-security-kubernetes-cross
     rerun_command: /test pull-security-kubernetes-cross
     run_if_changed: ^((build\/|hack\/lib\/).*)|(.*Makefile.*)$
-    skip_branches:
```

Comment:
This should not have been part of release cut change. It gets difficult to automate such special changes. Also difficult to track why they were removed and not rotated to a new version.